girderformindlogger/conf/girder.dist.cfg

```diff
@@ -9,7 +9,8 @@ replica_set = None
 
 [server]
 # Set to "production" or "development"
-mode = "production"
+mode = "development"
+# mode = "production"
```

Comment:
I don't think we want to commit this change to [`master`](https://github.com/ChildMindInstitute/mindlogger-app-backend/tree/master)